app/javascript/src/components/Dashboard/Contacts/Table/TableData.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+
+import ProfilePic from "images/ProfilePic";
+import { MenuHorizontal } from "neetoicons";
+import { Typography, Avatar, Dropdown } from "neetoui";
+
+export const contactsTableColumnData = setShowDeleteAlert => [
```

Comment:
So this file is not a React component. We use PascalCase only for naming React components or JavaScript classes.

Here this `contactsTableColumnData` is sort of a utility function. Thus move it to a `utils.js` within the closest scope.

Also, since we are moving it as a utility function, we should name it more util like, say as: `buildContactTableColumnData`

Reply:
Fixing it as per your recent loom video on slack. Thanks!